falcon/asgi/ws.py

```diff
@@ -512,6 +523,9 @@ class WebSocketOptions:
             unhandled error is raised while handling a WebSocket connection
             (default ``1011``). For a list of valid close codes and ranges,
             see also: https://tools.ietf.org/html/rfc6455#section-7.4
+        default_close_reasons (dict): A default mapping between the Websocket
+            close code and the reason why the connection is close. HTTPerrors
+            are not included and will be rendered automatically with HTTP status.
```

Comment:
```suggestion
            are not included and will be rendered automatically as HTTP status.
```

not sure what's the proper pronoun here. as sounded better, but I'm not a native English speaker so I'm probably wrong

Reply:
I guess my words are kind of confusing. Does the following expression makes better sense?

`Close codes corresponding to HTTPerrors are not included as they will be rendered automatically using HTTP status.`

Reply:
yes, that's better I think!